mlflow/server/js/src/utils/Utils.test.js

```diff
@@ -1,83 +1,83 @@
 import Utils from './Utils'
 
 test("formatMetric", () => {
-  expect(Utils.formatMetric(0)).toBe("0");
-  expect(Utils.formatMetric(0.5)).toBe("0.5");
-  expect(Utils.formatMetric(0.001)).toBe("0.001");
+  expect(Utils.formatMetric(0)).toEqual("0");
+  expect(Utils.formatMetric(0.5)).toEqual("0.5");
+  expect(Utils.formatMetric(0.001)).toEqual("0.001");
 
-  expect(Utils.formatMetric(0.12345)).toBe("0.123");
-  expect(Utils.formatMetric(0.12355)).toBe("0.124");
-  expect(Utils.formatMetric(-0.12345)).toBe("-0.123");
-  expect(Utils.formatMetric(-0.12355)).toBe("-0.124");
+  expect(Utils.formatMetric(0.12345)).toEqual("0.123");
+  expect(Utils.formatMetric(0.12355)).toEqual("0.124");
+  expect(Utils.formatMetric(-0.12345)).toEqual("-0.123");
+  expect(Utils.formatMetric(-0.12355)).toEqual("-0.124");
 
-  expect(Utils.formatMetric(1.12345)).toBe("1.123");
-  expect(Utils.formatMetric(1.12355)).toBe("1.124");
-  expect(Utils.formatMetric(-1.12345)).toBe("-1.123");
-  expect(Utils.formatMetric(-1.12355)).toBe("-1.124");
+  expect(Utils.formatMetric(1.12345)).toEqual("1.123");
+  expect(Utils.formatMetric(1.12355)).toEqual("1.124");
+  expect(Utils.formatMetric(-1.12345)).toEqual("-1.123");
+  expect(Utils.formatMetric(-1.12355)).toEqual("-1.124");
 
-  expect(Utils.formatMetric(12.12345)).toBe("12.12");
-  expect(Utils.formatMetric(12.12555)).toBe("12.13");
-  expect(Utils.formatMetric(-12.12345)).toBe("-12.12");
-  expect(Utils.formatMetric(-12.12555)).toBe("-12.13");
+  expect(Utils.formatMetric(12.12345)).toEqual("12.12");
+  expect(Utils.formatMetric(12.12555)).toEqual("12.13");
+  expect(Utils.formatMetric(-12.12345)).toEqual("-12.12");
+  expect(Utils.formatMetric(-12.12555)).toEqual("-12.13");
 
-  expect(Utils.formatMetric(123.12345)).toBe("123.1");
-  expect(Utils.formatMetric(123.15555)).toBe("123.2");
-  expect(Utils.formatMetric(-123.12345)).toBe("-123.1");
-  expect(Utils.formatMetric(-123.15555)).toBe("-123.2");
+  expect(Utils.formatMetric(123.12345)).toEqual("123.1");
+  expect(Utils.formatMetric(123.15555)).toEqual("123.2");
+  expect(Utils.formatMetric(-123.12345)).toEqual("-123.1");
+  expect(Utils.formatMetric(-123.15555)).toEqual("-123.2");
 
-  expect(Utils.formatMetric(1234.12345)).toBe("1234.1");
-  expect(Utils.formatMetric(1234.15555)).toBe("1234.2");
-  expect(Utils.formatMetric(-1234.12345)).toBe("-1234.1");
-  expect(Utils.formatMetric(-1234.15555)).toBe("-1234.2");
+  expect(Utils.formatMetric(1234.12345)).toEqual("1234.1");
+  expect(Utils.formatMetric(1234.15555)).toEqual("1234.2");
+  expect(Utils.formatMetric(-1234.12345)).toEqual("-1234.1");
+  expect(Utils.formatMetric(-1234.15555)).toEqual("-1234.2");
 
-  expect(Utils.formatMetric(1e30)).toBe("1e+30");
+  expect(Utils.formatMetric(1e30)).toEqual("1e+30");
 });
 
 test("formatDuration", () => {
```

Comment:
Changed these to toEqual because toBe is for object identity and may not always work depending on the JS engine implementation.